package.json

```diff
@@ -7,7 +7,12 @@
     },
     "devDependencies": {
         "axios": "^1.6.1",
+        "jquery": "^3.7.1",
```

Comment:
why add jQuery Bootstrap 5 does not need it.

> Bootstrap 5 is designed to be used without jQuery

https://getbootstrap.com/docs/5.0/getting-started/javascript/#still-want-to-use-jquery-its-possible